dvc/command/get.py

```diff
@@ -51,9 +51,7 @@ def _get_file_from_repo(self):
 
 
 def add_parser(subparsers, parent_parser):
-    GET_HELP = (
-        "Download file or directory from any DVC project or Git repository."
-    )
+    GET_HELP = "Download file or directory from any DVC or Git repository."
```

Comment:
Feels like `DVC or Git` has the same meaning as `DVC project or Git`. Maybe `Git repository w/ or w/o DVC` or something like that?

Reply:
A "DVC project" doesn't have to be in a Git repo but these commands require a Git repo.

I realize it's tricky wording but I'd prefer to mention DVC before Git. Maybe just "DVC repository or Git repo"? It's kind of repetitive though...

Reply:
Well, but that DVC repo has to be a git repo too, that is why I'm wondering if we could clarify that.

Reply:
`Git repository w/ or w/o DVC` does not sound right to me. Feels like we deal with Git in the first place. While we should emphasize that it's about DVC controlled files.

How about: `Download a Git or DVC controlled file or directory`?

Reply:
I think @jorgeorpinel point is that relying on DVC repository (has to be Git repo) vs DVC project (can be Git or not Git) is very hard to understand, very hard to maintain in docs. Phrases like `DVC project or Git repository` probably won't mean much for users.

Reply:
Hi guys, getting back to this finally.

> relying on DVC repository (has to be Git repo) vs DVC project (can be Git or not Git) is very hard to understand

Agreed.

> How about: Download a Git or DVC controlled file or directory?

Good option, updated to "tracked by DVC or by Git" for both commands in this PR and docs in iterative/dvc.org/pull/978

Reply:
Resolving this, but please feel free to participate in this related discussion Ruslan: https://github.com/iterative/dvc.org/pull/931#pullrequestreview-354892810

Thanks!